Development.md

```diff
@@ -0,0 +1,194 @@
+
+Cached versions of the `dev_bundle` are stored in the root directory of the checkout.  Keeping them around will prevent the need to re-download them when switching between branches, but they do become quite large as they collect so delete them as necessary!
+
+### Rebuilding the "Dev Bundle"
+
+Rebuilding requires a C and C++ compiler, `autotools`, and `scons`.
+
+To build everything from scratch (`node`, `npm`, `mongodb`, etc.) and re-package NPM modules, simply run the following script:
+
+```sh
+$ ./scripts/generate-dev-bundle.sh
+```
+
+This will generate a new tarball (`dev_bundle_<Platform>_<arch>_<version>.tar.gz`) in the root of the checkout.  Assuming you bumped the `BUNDLE_VERSION`, the new version will be extracted automatically when you run `./meteor`.  If you are rebuilding the same version (or didn't bump the version number), you should delete the existing `dev_bundle` directory to ensure the new tarball is extracted when you run `./meteor`.
+
+## Additional documentation
+
+The Meteor core is best documented within the code itself, however, many components also have a `README.md` in their respective directories.
+
+Some compartmentalized portions of Meteor are broken into packages ([see a list of packages](packages/)) and they almost all have a `README.md` within their directory.  For example, [`ddp`](packages/ddp/README.md), [`ecmascript`](packages/ecmascript/README.md) and [`tinytest`](packages/tinytest/README.md).
+
+For the rest, try looking nearby for a `README.md`.  For example, [`isobuild`](tools/isobuild/README.md) or [`cordova`](tools/cordova/README.md).
+
+## Tests
+
+### Test against the local meteor copy
+
+When running any of tests, be sure run them against the checked-out copy of Meteor instead of
+the globally-installed version.  This means ensuring that the command is `path-to-meteor-checkout/meteor` and not just `meteor`.
+
+This is important so that tests are run against the version in development and not the stable (installed) Meteor release.
+
+### Running tests on Meteor core
+
+When you are working with code in the core Meteor packages, you will want to make sure you run the
+full test-suite (including the tests you added) to ensure you haven't broken anything in Meteor. The
+`test-packages` command will do just that for you:
+
+    ./meteor test-packages
+
+Exactly in the same way that [`test-packages` works in standalone Meteor apps](https://guide.meteor.com/writing-atmosphere-packages.html#testing), the `test-packages` command will start up a Meteor app with [TinyTest](./packages/tinytest/README.md).  To view the results, just connect to `http://localhost:3000` (or your specified port) and view the results.
```

Comment:
Can remove the "and view the results." part of the last sentence (since "To view the results" is already mentioned at the beginning).